lib/cdo/contact_rollups_validation.rb

```diff
@@ -269,6 +269,28 @@ class ContactRollupsValidation
       min: 0,
       max: 0
     },
+    {
+      # Double-check consistency of the complex (but necessary) query
+      # to mark contacts in the Teacher role
+      name: "Check that all contacts that match 'Teacher Query' "\
+            "are in Teacher role",
+      query: "SELECT COUNT(*) from contact_rollups_daily
+              WHERE (dashboard_user_id is not null
+              OR forms_submitted LIKE '%BringToSchool2013%'
+              OR forms_submitted LIKE '%ClassSubmission%'
+              OR forms_submitted LIKE '%DistrictPartnerSubmission%'
+              OR forms_submitted LIKE '%HelpUs2013%'
+              OR forms_submitted LIKE '%K5OnlineProfessionalDevelopmentPostSurvey%'
+              OR forms_submitted LIKE '%K5ProfessionalDevelopmentSurvey%'
+              OR forms_submitted LIKE '%ProfessionalDevelopmentWorkshop%'
+              OR forms_submitted LIKE '%ProfessionalDevelopmentWorkshopSignup%'
+              OR forms_submitted LIKE '%StudentNomination%'
+              OR forms_submitted LIKE '%TeacherNomination%'
```

Comment:
Ideally this list could be defined in one place and reused from `FORM_KINDS_TEACHER`. I'd suggest at least mentioning that const in a comment (or in the name) so it's obvious where to look if this fails.